modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/ProfileEpisodeListViewModel.kt

```diff
@@ -57,9 +56,9 @@ class ProfileEpisodeListViewModel @Inject constructor(
     fun setup(mode: Mode) {
         this.mode = mode
         val episodeListFlowable = when (mode) {
-            is Mode.Downloaded -> episodeManager.findDownloadEpisodesFlow().asFlowable()
-            is Mode.Starred -> episodeManager.findStarredEpisodesRxFlowable()
-            is Mode.History -> episodeManager.findPlaybackHistoryEpisodesRxFlowable()
+            is Mode.Downloaded -> episodeManager.findDownloadEpisodesFlow()
+            is Mode.Starred -> episodeManager.findStarredEpisodesFlow()
+            is Mode.History -> episodeManager.findPlaybackHistoryEpisodesFlow()
```

Comment:
I converted these methods to flow for Wear OS, so it seems sensible to switch these at the same time.